src/libs/actions/BankAccounts.ts

```diff
@@ -294,6 +361,16 @@ function deletePaymentBankAccount(bankAccountID: number) {
         ],
     };
 
+    if (shouldClearIOULastUsedPaymentMethod) {
+        onyxData.successData?.push({
+            onyxMethod: Onyx.METHOD.MERGE,
+            key: ONYXKEYS.NVP_LAST_PAYMENT_METHOD,
+            value: {
+                [personalPolicy?.id]: null,
+            },
+        });
```

Comment:
just wanted to double check here if we also might need to reset `lastUsed` value.